lib/load.php

```diff
@@ -104,6 +104,7 @@ function gutenberg_is_experiment_enabled( $name ) {
 require dirname( __FILE__ ) . '/utils.php';
 
 require dirname( __FILE__ ) . '/full-site-editing.php';
+require dirname( __FILE__ ) . '/full-site-editing/default-template-types.php';
```

Comment:
The idea is to eventually move all FSE-related libraries into a `full-site-editing` folder, to make it all cleaner.